resources/js/Pages/User/Create.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <h1 class="text-2xl mb-4">Create User</h1>
+
+    <!-- User creation form -->
+    <form @submit.prevent="createUser">
+      <div class="mb-4">
+        <label for="name" class="block text-sm font-medium text-gray-700">Name</label>
+        <input type="text" v-model="user.name" id="name" class="mt-1 block w-full px-4 py-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent" />
+      </div>
+
+      <div class="mb-4">
+        <label for="email" class="block text-sm font-medium text-gray-700">Email</label>
+        <input type="email" v-model="user.email" id="email" class="mt-1 block w-full px-4 py-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent" />
+      </div>
+
+      <div class="mb-4">
+        <label for="password" class="block text-sm font-medium text-gray-700">Password</label>
+        <input type="password" v-model="user.password" id="password" class="mt-1 block w-full px-4 py-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent" />
+      </div>
+
+      <!-- Buttons for Submit and Cancel -->
+      <div class="flex justify-end">
+        <button type="submit" class="btn btn-primary mr-2">Create</button>
+        <InertiaLink :href="route('users.index')" class="btn btn-secondary">Cancel</InertiaLink>
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+import { InertiaLink } from '@inertiajs/inertia-vue3'
+
+export default {
+  props: {
+    user: Object,
+  },
+  components: {
+    InertiaLink,
+  },
+  data() {
+    return {
+      user: {
+        name: '',
+        email: '',
+        password: ''
+      }
+    };
+  },
+  methods: {
+    createUser() {
+      // Logic to handle user creation (e.g., make an API call)
+      this.$inertia.post(route('users.store'), this.user);
+    },
+  },
+};
+</script>
+
+<style scoped>
```

Comment:
I think that we can style this using tailwind CSS classes - there is no need to use scope style here

Reply:
Maybe create components for inputs, buttons and all elements that will be often use? Thanks to that we will have consistent design and programming views will be faster. Please talk about that with others.